packages/core/src/App/Containers/RealityCheckModal/reality-check-modal.jsx

```diff
@@ -3,25 +3,51 @@ import PropTypes from 'prop-types';
 import React from 'react';
 import { useHistory } from 'react-router-dom';
 import { Input } from '@deriv/components';
-import { routes, validNumber } from '@deriv/shared';
+import { getDecimalPlaces, routes, validNumber } from '@deriv/shared';
 import { localize } from '@deriv/translations';
 import { connect } from 'Stores/connect';
 import BriefModal from './brief-modal.jsx';
 import SummaryModal from './summary-modal.jsx';
+import SpendingLimitModal from './spending-limit-modal.jsx';
 
-const IntervalField = ({ values, touched, errors, handleChange, handleBlur }) => (
-    <div className='reality-check__fieldset'>
+const SpendingLimitIntervalField = ({ values, disabled, touched, errors, handleChange, handleBlur }) => (
+    <div className='reality-check__fieldset reality-check__fieldset--spending-limit'>
         <Field name='interval'>
             {({ field }) => (
                 <Input
                     {...field}
+                    name='max_30day_turnover'
                     data-lpignore='true'
                     type='text'
-                    label={localize('Time interval')}
+                    label={localize('My spending limit (EUR)')}
+                    value={values.max_30day_turnover}
+                    onChange={handleChange}
+                    onBlur={handleBlur}
+                    disabled={disabled}
+                    required
+                    error={touched.max_30day_turnover && errors.max_30day_turnover}
+                    autoComplete='off'
+                    maxLength='13'
+                />
+            )}
+        </Field>
+    </div>
+);
+
+const TradingViewIntervalField = ({ values, touched, errors, handleChange, handleBlur }) => (
+    <div className='reality-check__fieldset reality-check__fieldset--interval'>
+        <Field name='interval'>
+            {({ field }) => (
+                <Input
+                    {...field}
+                    name='interval'
+                    data-lpignore='true'
+                    type='text'
+                    label={localize('After … minutes of trading')}
```

Comment:
why do we have `...`? 😅

Reply:
@balakrishna-binary  this is what we have on Figma